processor/.classpath

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ignore .classpath using gitignore command
we do not want to check in these files.